CHANGELOG.md

```diff
@@ -1,6 +1,40 @@
 # Changelog
 
-## [1.5](https://github.com/arborx/arborx/tree/v1.5.0) (2023-12-16)
+## [1.6](https://github.com/arborx/arborx/tree/v1.6) (2024-04-11)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.5...v1.6)
+
+**Build Changes:**
+- Require Kokkos 4.1.00 [\#973](https://github.com/arborx/ArborX/pull/973)
+
+**New features:**
+- Introduce new API for ArborX indexes (EXPERIMENTAL) [\#970](https://github.com/arborx/ArborX/pull/970), [\#1017](https://github.com/arborx/ArborX/pull/1017)
+- Add Moving Least Squares (MLS) interpolation (EXPERIMENTAL) [\#946](https://github.com/arborx/ArborX/pull/946), [\#992](https://github.com/arborx/ArborX/pull/992), [\#1000](https://github.com/arborx/ArborX/pull/1000)
+
+**Enhancements:**
+- Introduce `PairValueIndex` and `Experimental::attach_indices` [\#969](https://github.com/arborx/ArborX/pull/969), [\#1016](https://github.com/arborx/ArborX/pull/1016), [\#1036](https://github.com/arborx/ArborX/pull/1036)
+- Improve the simple intersection example [\#1008](https://github.com/arborx/ArborX/pull/1008)
+- Improve the brute force example [\#1009](https://github.com/arborx/ArborX/pull/1009)
+- Auto-fetch Google benchmark when not found locally [\#1039](https://github.com/arborx/ArborX/pull/1039)
+- Allow default initialized distributed tree [\#1040](https://github.com/arborx/ArborX/pull/1040)
+- Implement distance point-triangle [\#1046](https://github.com/arborx/ArborX/pull/1046)
+- Implement nearest query for BruteForce [\#1053](https://github.com/arborx/ArborX/pull/1053)
+- Add helper functions to construct predicates [\#1038](https://github.com/arborx/ArborX/pull/1038)
+- Add an example of a distributed tree k-nearest neighbors search [\#724](https://github.com/arborx/ArborX/pull/724)
+- Improve the triangle intersection example [\#936](https://github.com/arborx/ArborX/pull/936)
```

Comment:
I would prefer if we merged L16 L17 and L24 into a single improve examples entry